src/dashboard/frontend/app/locale/translations.sv.po

```diff
@@ -463,6 +467,7 @@ msgstr ""
 "Kunde inte starta SIP; inga filer ordnade i arkivbeståndet \"{{title}}\"."
 
 #: app/archivesspace/archivesspace.controller.js:162
+#: app/arrangement/arrangement.controller.js:362
```

Comment:
Should we add `src/dashboard/frontend/app/locale/*.po` to `.gitattributes`?
See https://help.github.com/en/articles/customizing-how-changed-files-appear-on-github.

Reply:
Thanks for the reference. I added the `*.po` and `translation.json` to `.gitattributes`.